_topic_maps/_topic_map.yml

```diff
@@ -3098,6 +3098,8 @@ Topics:
     File: virt-live-migration-limits
   - Name: Migrating a virtual machine instance to another node
     File: virt-migrate-vmi
+  - Name: Migrating a virtual machine on a dedicated secondary network
```

Comment:
Better rephrase to: `Migrating a virtual machine **via/through** dedicated secondary network`

Reply:
Yes, that will also suite